ebi/commands/bgdeploy.py

```diff
@@ -28,18 +28,59 @@ def get_environ_name_for_cname(app_name, cname):
     raise ValueError('Could not find environment for applied app_name and cname')
 
 
-def get_primary_env_capacity(group_name):
+def get_instance_health(group_name, number):
     autoscale = boto3.client('autoscaling')
     as_json = autoscale.describe_auto_scaling_groups(
         AutoScalingGroupNames=[group_name])
+    instances = as_json['AutoScalingGroups'][0]['Instances']
+    instance_number = len(instances)
+    if instance_number != number:
+        return False
+
+    for instance in instances:
+        # Wait for the all of instance status to be healthy.
+        ec2 = boto3.client('ec2')
+        instance = ec2.describe_instance_status(InstanceIds=[instance['InstanceId']])
+        if not instance['InstanceStatuses']:
```

Comment:
question: InstanceStatuses key is always expected? Or is it better to use instance.get('InstanceStatuses')? I'm considering a case of key error.